src/main/java/Villager.java

```diff
@@ -0,0 +1,16 @@
+import java.util.List;
+
+public class Villager implements Tradable, Domesticatable {
+
```

Comment:
Maybe should add documentation for the Villager class

Reply:
done